### PR TITLE
Homework 4

### PR DESCRIPTION
**Pull request by mgovorovsky**
Hello, Denis!
Please check my previous homework and allow me to merge  the branches.
Thank you in advance!

### CODE REVIEW

**Comment by mgovorovsky** [comment]
Hello, Denis!
Last h/w has been added, it will be updated till next lesson. 
Please allow to merge all changes. 
Thanks in advance.

File: homework_6.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+def clear():
+    if name == 'nt':
+        _ = system('cls')
+    else:
+        _ = system('clear')
+
+
+n = 1
+while n != 0:
```
Comment (dvk-net) [inline homework_6.py:116]: while True:

why n was introduced at all?

File: homework_6.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        i = i + 1
+
+
+def get_current_time():
+    date_now = datetime.datetime.now()
+    hours = int(date_now.strftime("%H"))
+    minutes = int(date_now.strftime("%M"))
+    seconds = int(date_now.strftime("%S"))
+
+    h2 = hours % 10
```
Comment (dvk-net) [inline homework_6.py:97]:
```python
time = date_now.strftime("%H%M%S") # time -> "121343"
h1 = time[0]
h2 = time[1] # etc
# or
h1, h2, m1, m2, s1, s2 = time
```

File: homework_6.py (hunk 1 above)
Comment (dvk-net) [inline homework_6.py:116]:
```python
while True:
    pass
```

**Review by dvk-net — APPROVED** [review]
good /10

**Pull request merged**